plugin.json

```diff
@@ -64,6 +64,6 @@
         "default": ""
       }
     ],
-    "footer": "Use this webhook URL to set up the Jira integration. See [documentation](https://about.mattermost.com/default-jira-plugin) to learn more.\n\n`https://SITEURL/plugins/jira/webhook?secret=WEBHOOKSECRET&team=TEAMURL&channel=CHANNELURL`"
+    "footer": "Use this webhook URL to set up the Jira integration. See [documentation](https://about.mattermost.com/default-jira-plugin) to learn more.\n\n`https://SITEURL/plugins/jira/webhook?secret=WEBHOOKSECRET`"
```

Comment:
I think it's using a different endpoint in the form

```
https://SITEURL/plugins/jira/api/v2/webhook?secret=WEBHOOKSECRET
```

Any value in having the old webhook URL format, or just keep it in the doc?

Reply:
I've added the 2.0 and 2.1 webhook formats so it's obvious something has changed.  In the docs, I'm going to reference 2.0 setup (archived) and also 2.1's webhook setup.

Reply:
Thanks! The changes here will be reflected for v2.1 only (since v2.0 already shipped), so I'm wondering if we need to call out the specific Jira version in this case

@aaronrothschild